x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/bulk/TransportBulkShardOperationsAction.java

```diff
@@ -62,7 +62,9 @@ public TransportBulkShardOperationsAction(
         return shardOperationOnPrimary(request.shardId(), request.getOperations(), primary, logger);
     }
 
-    static WritePrimaryResult<BulkShardOperationsRequest, BulkShardOperationsResponse> shardOperationOnPrimary(
+    // TODO: move the bulk shard ops api classes to action package? Then the visibility of this method can be
+    // package proctected instead of public.
```

Comment:
I am fine with this.

Reply:
Thanks for checking. I removed the TODO: b27e739709ffd31ca785e40bd9521094bf126915